src/mpid/ch4/netmod/ofi/ofi_pre.h

```diff
@@ -129,17 +129,27 @@ typedef struct {
         uint64_t lmt_cntr;
```

Comment:
The commit message for `ch4/ofi: make sure am message buffers are consecutive` has some typos.

It should say:

```
ch4/ofi: make sure am message buffers are consecutive

When fi_sendv is not available, if the whole message fits in the header
buffer space, we can directly send it as a single buffer.

When the message doesn't fit in the header space, we'll have to allocate
a separate send buffer. We can reuse the pack_buffer pointer since we
won't need both.

Move the header pool macros to ofi_pre.h since its context is relevant
there.
```

Reply:
Fixed. For the record, I never liked the "spot the difference game" -- I only found `int` -> `in`.

Reply:
The word before went from `fit` to `fits`.

Reply:
> 
> 
> That's why I copied the whole thing in so you could copy/paste it. 😄

Oh, I was dense 🤭